pkg/distributor/otel.go

```diff
@@ -401,8 +401,8 @@ func otelMetricsToMetadata(addSuffixes bool, md pmetric.Metrics) []*mimirpb.Metr
 				metric := scopeMetrics.Metrics().At(k)
 				entry := mimirpb.MetricMetadata{
 					Type: otelMetricTypeToMimirMetricType(metric),
-					// TODO(krajorama): when UTF-8 is configurable from user limits, replace "false" appropriately.
-					MetricFamilyName: prometheustranslator.BuildCompliantName(metric, "", addSuffixes, false),
+					// TODO(krajorama): when UTF-8 is configurable from user limits, use BuildCompliantName. See https://github.com/prometheus/prometheus/pull/15664
```

Comment:
```suggestion
					// TODO(krajorama): when UTF-8 is configurable from user limits, use BuildMetricName. See https://github.com/prometheus/prometheus/pull/15664
```

Small typo on the function that creates metrics with UTF-8

https://github.com/prometheus/prometheus/pull/15664/files#diff-29ec8315e06d13c5ff44d68a3efa4efc5dda7c0432b7d2c07634d9d852ae34f0R244